projects/registry-data-quality.md

```diff
@@ -0,0 +1,103 @@
+([based on Google Analytics Data](https://lookerstudio.google.com/s/qhPHQtyfRbU)).
+
+In the last months SIG Comms implemented a set of changes to the registry, to make it more user-friendly and easier to find:
+
+- [A semi-automatic semi-complete scanner](https://github.com/open-telemetry/opentelemetry.io/pull/1920)
+- [Rework of the design](https://github.com/open-telemetry/opentelemetry.io/pull/3730), including a new search engine, badges, quick installation instructions
+- [A daily run workflow that keeps package versions up-to-date](https://github.com/open-telemetry/opentelemetry.io/pull/3840)
+- [A JSON schema definition to verify registry entry format](https://github.com/open-telemetry/opentelemetry.io/pull/4805)
+- [Included the registry in many pages, including the sidebar navigation for all languages and the collector](https://github.com/open-telemetry/opentelemetry.io/pull/3932)
+
+Overall, the registry is in a good shape and is slowly turning into a place that people use for it's original intend.
+
+### Current challenges
+
+While being reworks in incremental steps in the last few months, the registry is still facing a set of challenges, that require a bigger
+review and eventually a collaboration across many SIGs to be addressed:
+
+1. Besides the update of package versions, the maintenance is a manual process:
+   1. Packages from different project repositories are either detected by a semi-automatic scanner, that needs to be run
+      manually, or by SIGs reporting new entries themselves.
+   2. After a package has been added to the registry, it is a manual process to verify if the package is still available and
+      if the meta-data is still correct.
+2. Packages lack a lot of information, and the information available is sometimes of bad quality. Metadata that might be available
+   for packages is not in a human readable format, or in the case of the collector (`mdatagen`-data) not consumed by the registry.
+3. [Vendors](https://opentelemetry.io/ecosystem/vendors/), [Integrations](https://opentelemetry.io/ecosystem/integrations/), [Adopters](https://opentelemetry.io/ecosystem/adopters/) and [Distributions](https://opentelemetry.io/ecosystem/distributions/) are separated out into their own pages, and entries on these pages are also hard to maintain and verify.
+4. Adding an entry to the registry requires the addition of a YAML file, in a pre-defined format, that until recently has not been enforced consistently.
+
+All these issues are mostly related to the quality of the data, i.e. what is available, what is outdated, how detailed is the data
+and how useable is that data. If we address those issues end users will see the registry as a valuable place and will use
+it to find the building blocks they need, and by reaching that state, we can use the registry to accomplish other goals, like
+
+- making end users aware of non-otel-community created components they can use. Combined with a system to label "good quality"
```

Comment:
I think this is another whole project of its own :smile:

Reply:
Agreed, this project would be the foundation for such another project